stock_lot_suffix/models/stock_lot.py

```diff
@@ -22,3 +22,12 @@ def name_get(self):
                 new_name += "-" + rec.channel_category
             new_name_list.append((name[0], new_name))
         return new_name_list
+
+    @api.depends("analytic_account_ids")
+    def _compute_lot_suffix(self):
+        for rec in self:
+            rec.lot_suffix = False
+            # We assume that there is only one analytic account with lot_suffix if any.
+            analytic_account = rec.analytic_account_ids.filtered(lambda x: x.lot_suffix)
+            if analytic_account:
+                rec.lot_suffix = analytic_account[0].lot_suffix
```

Comment:
To be slightly shorter.
```suggestion
            # We assume that there is only one analytic account with lot_suffix if any.
            aa = rec.analytic_account_ids.filtered(lambda x: x.lot_suffix, limit=1)
            rec.lot_suffix = aa.lot_suffix if aa else False
```

Reply:
I don't think there is `limit` attribute for `filtered`.
Even we use like `rec.analytic_account_ids.filtered(lambda x: x.lot_suffix)[0]`, will throw error if there is no accounts.

Reply:
Then, `rec.analytic_account_ids.filtered(lambda x: x.lot_suffix)[:1]`?